src/OrchardCore.Themes/TheAdmin/Assets/js/darkmode.js

```diff
@@ -0,0 +1,23 @@
+var darkmode = (darkmode === undefined) ? false : darkmode;
+
+$(function () {
+    if ($('body').hasClass('darkmode') || darkmode)
+    {
+        $('#admin-css').attr('href', $('#admin-css').attr('href').replace('/TheAdmin.css', '/TheAdmin-darkmode.css'));
```

Comment:
Longer term we will want a way for these user preferences to be stored against the User property rather than in cookies, and for this to be a `Condition` applied to the resource tag. 
But I would not block the feature on this.

Reply:
Last thing I want to add is the automatic selection of the dark mode if your OS is on dark mode.
Will work on small details like these today then I will merge it when I think it's solid enough.

Reply:
I think you meant User IEntity.Properties? Right now I use LocalStorage of the browser which cannot be read from server side. Maybe it should use a Cookie for all those AdminPreferences instead. 

Not sure what you mean with the Condition applied on the resource tag and what it would be for.